jupyter_server_proxy/handlers.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, *args, **kwargs):
         self.host_allowlist = kwargs.pop('host_allowlist', ['localhost', '127.0.0.1'])
         self.rewrite_response = kwargs.pop(
             'rewrite_response',
-            lambda host, port, path, response: response.body
+            lambda request, host, port, path, response: {}
```

Comment:
It makes me really nervous to see a [mutable object inside a default argument](https://www.google.com/search?q=python+mutable+default+argument). But in this case I think it's effectively immutable thanks to the `lambda`.